ldk/javascript/src/whisper/types.ts

```diff
@@ -313,6 +324,8 @@ export type ChildComponents =
   | RadioGroup
   | Select
   | SectionTitle
+  | DateTimeInput
```

Comment:
Same in here you dont need to duplicate DateTimeInput its already presented on the line 314.

Reply:
@RusGraf I have updated the code and get your comments solved :)